test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
Let me raise another concern: If on a machine the `defaultAxesColorOrder` differs from `get(0,'Factory')` (e.g., due to `startup.m`), it wouldn't matter on the current implementation, but would fail if I would remove that line.
Is this a reason to keep it after all or is this a precondition for a `m2t` developer to not modify graphical root?